src/WasmDis.ts

```diff
@@ -448,6 +454,17 @@ export class WasmDisassembler {
     this._lines.push(this._buffer);
     this._buffer = '';
   }
+  private logStartOfBodyFunctionOffset() {
+    if (this.addOffsets) {
+      this._currentBodyFunctionOffset = [this._currentPosition];
+    }
+  }
+  private logEndOfBodyFunctionOffset() {
+    if (this.addOffsets && this._currentBodyFunctionOffset.length == 1) {
```

Comment:
Optional: Instead of checking if the length is 1, check for its existence, then set it back to null.

Reply:
done